<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}
QComboBox{
	background-color: rgb(245, 245, 245);
	font: 12pt &quot;Times New Roman&quot;;
}

QLineEdit{
	background-color: rgb(245, 245, 245);
	font: 12pt &quot;Times New Roman&quot;;
}

QLineEdit:hover{
	background-color: rgb(255, 255, 255);
}

QLabel{
	font: 75 12pt &quot;Times New Roman&quot;;
	
}

QRadioButton{
	font: 9pt &quot;Times New Roman&quot;;
}

QPushButton{
	border-radius: 6px;
}
</string>
   </property>
   <widget class="QLabel" name="partLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Part #</string>
    </property>
   </widget>
   <widget class="QLabel" name="sheenLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Sheen</string>
    </property>
   </widget>
   <widget class="QLabel" name="brandLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="packageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Package</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="vendorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Vendor</string>
    </property>
   </widget>
   <widget class="QFrame" name="top_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color:rgba(0, 51, 162, 255);
}

QLabel{
	background-color: none;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Insert a New Product</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commitButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: none;
	border-color: rgb(0, 0, 0);
	color: rgb(54, 54, 54);
	font: 87 10pt &quot;Arial Black&quot;;
	border: 1px solid;
}

QPushButton:hover{
	
	background-color: rgb(175, 232, 162);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: none;
	border-color: rgb(0, 0, 0);
	color: rgb(54, 54, 54);
	font: 87 10pt &quot;Arial Black&quot;;
	border: 1px solid;
}

QPushButton:hover{
	background-color: rgb(221, 153, 153);
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_part">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_brand">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_color">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_units">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vendor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_location">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_description">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sheen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_package">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>partLabel</zorder>
   <zorder>sheenLabel</zorder>
   <zorder>brandLabel</zorder>
   <zorder>packageLabel</zorder>
   <zorder>colorLabel</zorder>
   <zorder>unitLabel</zorder>
   <zorder>vendorLabel</zorder>
   <zorder>locationLabel</zorder>
   <zorder>descriptionLabel</zorder>
   <zorder>commitButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>top_bar</zorder>
   <zorder>lineEdit_part</zorder>
   <zorder>lineEdit_brand</zorder>
   <zorder>lineEdit_color</zorder>
   <zorder>lineEdit_units</zorder>
   <zorder>lineEdit_vendor</zorder>
   <zorder>lineEdit_location</zorder>
   <zorder>lineEdit_description</zorder>
   <zorder>comboBox_sheen</zorder>
   <zorder>comboBox_package</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_part</tabstop>
  <tabstop>comboBox_sheen</tabstop>
  <tabstop>lineEdit_brand</tabstop>
  <tabstop>comboBox_package</tabstop>
  <tabstop>lineEdit_color</tabstop>
  <tabstop>lineEdit_units</tabstop>
  <tabstop>lineEdit_vendor</tabstop>
  <tabstop>lineEdit_location</tabstop>
  <tabstop>lineEdit_description</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>commitButton</tabstop>
  <tabstop>widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
